### PR TITLE
[s] sharpening blocks now function the way god intended

### PR DESCRIPTION
**Pull request by PKPenguin321**
they can sharpen any object and can sharpen the same object more than once

### CODE REVIEW

**Comment by Zilenan91** [comment]
quickly merge it to master before anyone sees

File: code/game/objects/items/weapons/sharpener.dm [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	var/increment = 4
 	var/max = 30
```
Comment (Supermichael777) [inline code/game/objects/items/weapons/sharpener.dm]: you missed a spot

File: code/game/objects/items/weapons/sharpener.dm [hunk 2]
```diff
@@ -7,7 +7,7 @@
 	var/increment = 4
```
Comment (Supermichael777) [inline code/game/objects/items/weapons/sharpener.dm]: wasent this like ten or something beforehand?

File: code/game/objects/items/weapons/sharpener.dm [hunk 3]
```diff
@@ -4,10 +4,10 @@
 	icon_state = "sharpener"
 	desc = "A block that makes things sharp."
 	var/used = 0
-	var/increment = 4
-	var/max = 30
+	var/increment = 5
+	var/max = 15
```
Comment (GunHog) [inline code/game/objects/items/weapons/sharpener.dm]: > Adding balance changes to a joke pull

Comment (PKPenguin321) [inline code/game/objects/items/weapons/sharpener.dm]: Trust me

**Pull request merged**